commands/miscellaneous/getReactions.js

```diff
@@ -3,37 +3,38 @@ const { SlashCommandBuilder } = require('@discordjs/builders');
 
 // TODO: needs permissions 'MANAGE_CHANNELS', 'MANAGE_ROLES'
 module.exports = {
-    data: new SlashCommandBuilder()
-        .setName('getreactions')
-        .setDescription('Get reactions from a message')
-        .addStringOption(option =>
-            option.setName('message_id')
-                .setDescription('The message id you would like to get reactions from')
-                .setRequired(true))
-        .addChannelOption(option =>
-            option.setName('channel')
-                .setDescription('The channel this message is in')
-                .setRequired(true)),
-    async execute(interaction) {
-        const options = interaction.options;
-        const messageId = options.getString('message_id');
-        const channel = options.getChannel('channel');
+  data: new SlashCommandBuilder()
+    .setName('getreactions')
+    .setDescription('Get reactions from a message')
+    .addStringOption(option =>
+      option.setName('message_id')
+        .setDescription('The message id you would like to get reactions from')
+        .setRequired(true))
+    .addChannelOption(option =>
+      option.setName('channel')
+        .setDescription('The channel this message is in')
+        .setRequired(true))
+    .setDefaultPermission(false),
```

Comment:
This line is the only significant change, the rest is due to changing spacing from 4 spaces to 2 spaces.